group-community.md

```diff
@@ -137,6 +144,10 @@ Basically, at this point, the options are two:
 1. Everybody in the internet can see the list of hospitality exchange members
 2. Group is private, but the people's hospex data are public. ([this doesn't work if the group is hosted on CSS](https://github.com/CommunitySolidServer/CommunitySolidServer/issues/1442))
 
+[comment]: <> (What if we had a functional webId for each hospex community, and we created a public Solid Group that included them and authorized only members of this group to see people's hospex data? Each community would have a separate app/webpage, with their own webId. Only app would be authorized to access hospex data of members of the community it serves. The members directly or other apps/communities would not be authorized to read that data.
```

Comment:
Interesting, let's discuss this further!

I don't fully grasp: so all the hospex data would get proxied through the community? E.g. community would fetch the data (because it has the permission to read) and then it would pass the data further to the end user?

Reply:
I like a lot the idea that an organization could have its own id. Possibly `https://id.sleepy.bike` ... :) There was a [piece about webIds at the July (2022) Solid World](https://vimeo.com/731471028#t=43m25s)...

How we could and wanted to use this id, that's for further discussion - i haven't properly thought through the possibilities....